.github/actions/kots-e2e/action.yml

```diff
@@ -109,3 +124,10 @@ runs:
         aws-access-key-id: ${{ inputs.aws-access-key-id }}
         aws-secret-access-key: ${{ inputs.aws-secret-access-key }}
         aws-region: ${{ inputs.aws-region }}
+
+    - name: Delete GKE cluster if created
+      if: ${{ always() && inputs.platform == 'gke' }}
+      uses: ./.github/actions/kots-gke
+      with:
+        service-key: inputs.gke-service-key
+        delete: 'true'
```

Comment:
how about having two separate scripts, one for creating a gke cluster, and the other for deleting a cluster?

Reply:
e.g. `kots-gke-create` and `kots-gke-delete`

Reply:
good idea, they're split now. They both go through the gcloud authentication and kubeconfig generation process even though in the current setup the delete does not need it, since it's in the same workflow as the create, but this will give a little more flexibility for the future and that part only takes seconds